<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameInterface</class>
 <widget class="QWidget" name="GameInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>744</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>995</width>
    <height>744</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>995</width>
    <height>744</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>1</y>
     <width>141</width>
     <height>84</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>5</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>1</y>
     <width>74</width>
     <height>84</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>$</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>90</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="buy_new_guppy">
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>84</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/image/image/buy_new_guppy.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buy_new_guppyMom">
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>84</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/image/image/buy_new_guppyMom.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buy_small_carnivorous_fish">
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>84</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/image/image/default_toolbar_icon.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buy_large_carnivorous_fish">
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>84</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/image/image/default_toolbar_icon.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="upgrade_food">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>93</width>
            <height>84</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>93</width>
            <height>84</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/image/image/default_toolbar_icon.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="num_of_food">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>93</width>
            <height>84</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>93</width>
            <height>84</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/image/image/default_toolbar_icon.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buy_egg">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>93</width>
            <height>84</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>93</width>
            <height>84</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/image/image/buy_egg_1.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>884</x>
     <y>1</y>
     <width>111</width>
     <height>84</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/menu.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
